app/gameengine.py

```diff
@@ -3395,6 +3438,16 @@ def archive_hand_continuation():
                 prevent_life_loss = effect["prevent_life_loss"]
                 self.deal_damage(source_player, target_player, dealing_card, target_card, amount, special, prevent_life_loss, {}, self.continue_resolving_effects)
                 passed_on_continuation = True
+            case EffectType.EffectType_DealDamagePerStacked:
+                match effect.get("limitation"):
+                    case "center":
+                        holomems = effect_player.center
+                    case _:
+                        holomems = effect_player.get_holomem_on_stage()
+                num_of_stacked_cards = len([card for holomem in holomems for card in holomem["stacked_cards"] if is_card_holomem(card)])
+                effect["amount"] *= num_of_stacked_cards
```

Comment:
Don't do this. 
I'm pretty sure I eventually made a deep copy of the effect, but I'm traumatized about code actually changing the definitions.
Please make a deep copy of the effect here if you want to reuse this effect like this.